lib/errors/error-collector.js

```diff
@@ -288,10 +360,20 @@ class ErrorCollector {
 
     if (exception.error) {
       logger.trace(exception.error, 'Got exception to trace:')
+    } else {
+      logger.trace(transaction, 'Got transaction error to trace:')
     }
 
-    const errorTrace = createError(transaction, exception, this.config)
+    return true
+  }
 
+  /**
+   * Helper method for recording metrics about errors depending on the type of error that happened
+   *
+   * @param {Array} errorTrace list of error information
+   * @param {Transaction} transaction the transaction associated with the trace
+   */
+  _maybeRecordErrorMetrics(errorTrace, transaction) {
```

Comment:
line 384 and 385 can be combined right? I thought we had a rule eslint rule to catch that

Reply:
So we do have a rule, but it only kicks in if the logic doesn't change from the combining, in this case, the logic would change in that we would start logging "other" error metrics if the transaction doesn't exist. All that being said, I think it would be safe to combine regardless. Thoughts?

Reply:
never mind. this should stay. other has a special meaning within a transaction